include/zephyr/bluetooth/bluetooth.h

```diff
@@ -1417,6 +1417,20 @@ struct bt_le_per_adv_sync_param {
  */
 uint8_t bt_le_per_adv_sync_get_index(struct bt_le_per_adv_sync *per_adv_sync);
 
+/**
+ * @brief Get a periodic advertising sync object from the array index.
+ *
+ * This function is to get the periodic advertising sync object from
+ * the array index.
+ * The array has CONFIG_BT_PER_ADV_SYNC_MAX elements.
+ *
+ * @param index The index of the periodic advertising sync object.
+ *              The range of the index value is 0..CONFIG_BT_PER_ADV_SYNC_MAX-1
+ *
+ * @return The periodic advertising sync object of the array index or NULL if invalid index.
+ */
+struct bt_le_per_adv_sync *bt_le_per_adv_sync_get_from_index(uint8_t index);
```

Comment:
This is very similar to `bt_le_per_adv_sync_lookup_addr`, and I would suggest to use the same naming scheme, i.e.:

```suggestion
struct bt_le_per_adv_sync *bt_le_per_adv_sync_lookup_index(uint8_t index);
```

That would also make it similar to `bt_conn_lookup_index` which works very similar to this (although it is an internal function).

Reply:
Much better indeed, updated.